cwms-data-api/src/test/resources/cwms/cda/data/dto/forecast/forecast_spec_test.json

```diff
@@ -0,0 +1,12 @@
+{
+  "spec-id": "spec",
```

Comment:
This is missing the "designator" field

Reply:
That's not a field I have in the DTO. Would that be a string?

Reply:
yeah, that's the comment on the schema PR that needs to be added. It will be used to represent the forecast run